lib/loader/loader.cpp

```diff
@@ -127,7 +127,18 @@ Loader::parseModule(const std::filesystem::path &FilePath) {
 /// Parse module from byte code. See "include/loader/loader.h".
 Expect<std::unique_ptr<AST::Module>>
 Loader::parseModule(Span<const uint8_t> Code) {
-  auto Mod = std::make_unique<AST::Module>();
+  /// Filter out the MacOS or Linux AOT compiled WASM.
+  std::vector<Byte> ELFMagic = {0x7F, 0x45, 0x4C, 0x46};
+  std::vector<Byte> MacMagic = {0xCF, 0xFA, 0xED, 0xFE};
+  if (Code.size() >= 4 &&
+      (std::equal(ELFMagic.begin(), ELFMagic.end(), Code.begin()) ||
+       std::equal(MacMagic.begin(), MacMagic.end(), Code.begin()))) {
+    spdlog::error(ErrCode::MalformedMagic);
+    spdlog::error("    For the AOT compiled WASM loading, WasmEdge only "
```

Comment:
Since we have universal wasm format (wasm + AOT compiled shared library as a custom section). This error message is not correct.

Reply:
Do you have any suggestion about the error sentence?

Reply:
fixed.